Mathlib/LinearAlgebra/TensorProduct/Vanishing.lean

```diff
@@ -94,6 +94,7 @@ theorem sum_tmul_eq_zero_of_vanishesTrivially (hmn : VanishesTrivially R m n) :
   rw [Finset.sum_comm]
   simp_rw [← tmul_smul, ← smul_tmul, ← sum_tmul, h₂, zero_tmul, Finset.sum_const_zero]
 
+set_option maxHeartbeats 0
```

Comment:
Uuh does `0` mean `infty`? And if that is the case, do we really want that value instead of something actual large number?

Reply:
Oops, I think this is a stray merge, and can just be removed!

Reply:
Yes, it's not needed at all.